diff -Nur libcap/progs/Makefile libcap_patch/progs/Makefile
--- libcap/progs/Makefile	2015-03-12 09:30:15.313213680 +0900
+++ libcap_patch/progs/Makefile	2015-04-07 16:28:24.510784418 +0900
@@ -19,7 +19,7 @@
 all: $(BUILD)
 
 $(BUILD): %: %.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $< $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -pie -o $@ $< $(LDLIBS)
 
 %.o: %.c $(INCS)
 	$(CC) $(IPATH) $(CFLAGS) -c $< -o $@
